rskj-core/src/main/java/org/ethereum/net/rlpx/FrameCodecHandler.java

```diff
@@ -81,13 +81,11 @@ protected void encode(ChannelHandlerContext ctx, FrameCodec.Frame frame, ByteBuf
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
         if (channel.isDiscoveryMode()) {
-            loggerNet.debug("FrameCodec failed: ", cause);
+            loggerNet.debug("FrameCodec failed: address {}", ctx.channel().remoteAddress(), cause);
+        } else if (cause instanceof IOException) {
+            loggerNet.info("FrameCodec failed: address {}", ctx.channel().remoteAddress(), cause);
         } else {
-            if (cause instanceof IOException) {
-                loggerNet.info(String.format("FrameCodec failed: %s", ctx.channel().remoteAddress()), cause);
-            } else {
-                loggerNet.error("FrameCodec failed: ", cause);
-            }
+            loggerNet.warn("FrameCodec failed: ", cause);
```

Comment:
I would use different messages for each situation, other than that we would  see `FrameCodec failed` on different levels and it'll be hard do identify what's the context of the logging.

Reply:
There are 3 different levels, you can identify it with that. I just added some extra data.